vuepress/docs/next/tutorials/solution/strapi.md

```diff
@@ -0,0 +1,183 @@
+---
+sidebarDepth: 2
+---
+
+# Strapi PBC
+## Overview
+
+Entando 7.1 offers Strapi integration to provide the flexibility and customization of a leading open source and headless CMS. Users are able to create and organize application content through the seamless incorporation of Strapi into the App Builder experience. 
+
+The configuration of a Strapi instance and content are achieved using the Strapi packaged business capability (PBC) available on the Entando Cloud Hub. This tutorial covers:
+
+1. [Prerequisites](#prerequisites)
+2. [Installation](#installation)
+3. [Configuration](#configuration)
+4. [Next Steps](#next-steps)
+
+## Prerequisites
+
+- [A working instance of Entando](../../../docs/getting-started/)
+- Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+## Installation
+
+Entando's Strapi implementation is available from the Entando Cloud Hub via 3 bundles, which must be installed in the Local Hub of the App Builder.
+
+1. Within your Entando instance, create a persistent volume claim with the following properties: 
+   - Name: `pn-fe9131bb-ca5e5232-entando-strapi-server-pvc`
+   - Storage size: 2GB
+
+2. Using the ent CLI, apply the Custom Resource Definitions for the Strapi bundles:
+
+```
+ent ecr deploy --repo=https://github.com/Entando-Hub/entando-strapi-bundle.git
+ent ecr deploy --repo=https://github.com/Entando-Hub/entando-strapi-config-bundle.git
+ent ecr deploy --repo=https://github.com/Entando-Hub/entando-strapi-widgets-bundle.git
+```
+
+3. Install the bundles into your Local Hub:
+
+```
+ent ecr install entando-strapi-bundle
+ent ecr install entando-strapi-config-bundle
+ent ecr install entando-strapi-widgets-bundle
+```
+
+### Roles
+
+#### Entando Strapi Config Application
+
+All REST APIs are public. No Keycloak role is required.
+
+#### Entando Strapi Content Template Application
+
+All REST APIs are private and require the admin role. To add Keycloak role mapping for the `entando-strapi-config` and `entando-strapi-templates` clients:
```

Comment:
I might remove the first sentence. It seems like an internal detail and it could easily change as the impl evolves.